kubernetes/cray-opa/tests/opa/keycloak-admin_test.rego.tpl

```diff
@@ -158,6 +158,25 @@ test_unauth_role {
 
 }
 
+# Multi Tenancy Tests
+
+test_tenat_admin {
+
+  # Check the tenant admin only has access to tenant endpoints
+  allow.http_status == 403 with input as {"attributes": {"request": {"http": {"method": "GET", "path": "/apis/api1", "headers": {"authorization": "Bearer {{ .tenantToken }}", "tenant": "vcluster-coke"}}}}}
```

Comment:
A nit, but while we talk about 'coke' vs. 'pepsi' conceptually, suggest we make the mock tenant names generic (not specifying a brand!).